### PR TITLE
Feat/add custom hook

### LINKED ISSUE #34: Add default hooks
Add a couple of common hooks (localStorage?) to define folder structure.
 - Implement the hygen generator for hooks

### PR DESCRIPTION
**Pull request by jgleal**
### Describe your changes:

Added a custom hook to manage local storage

closes #34

### Checklist before requesting a review:

- [x] Your code builds clean without any errors or warnings
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: src/components/Technologies/Technologies.test.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ test('renders Technologies component', () => {
   const technologiesComponent = render(
     <Technologies />
   )
-  const iconsAlt = ['Vite Logo', 'React Logo', 'Typescript Logo', 'Playwright Logo', 'Storybook Logo', 'Hygen Logo', 'Stitches Logo']
+  const iconsAlt = ['Vite Logo', 'React Logo', 'Typescript Logo', 'Playwright Logo', 'Hygen Logo', 'Stitches Logo']
```
Comment (robertoHeCi) [inline src/components/Technologies/Technologies.test.tsx:10]: Should we change the Stitches by the Styled one?

File: src/hooks/useStorage.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { useCallback, useState, useEffect } from 'react'
+
+export function useLocalStorage (key:string, defaultValue:string) {
```
Comment (robertoHeCi) [inline src/hooks/useStorage.tsx:3]: What if we add a test for this hook? 🤔

**Review by robertoHeCi — COMMENTED** [review]
I’ve left just a couple of comments ;)

File: src/hooks/useStorage.tsx (hunk 2 above)
Comment (jgleal) [reply src/hooks/useStorage.tsx:3]: a what? okok, I'll do it.... the rush

File: src/components/Technologies/Technologies.test.tsx (hunk 1 above)
Comment (jgleal) [reply src/components/Technologies/Technologies.test.tsx:10]: Stitches are still present in this PR.

**Review by Donivanes — APPROVED** [review]

**Review by robertoHeCi — APPROVED** [review]

**Pull request merged**